browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3596,6 +3599,15 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_BRAVE_WALLET_PORTFOLIO_ASSET_NETWORK_DESCRIPTION" desc="Portfolio asset item network description">
       <ph name="SYMBOL"><ex>BAT</ex>%1$s</ph> on <ph name="NETWORK"><ex>Ethereum Mainnet</ex>%2$s</ph>
     </message>
+    <message name="IDS_NO_NFTS" desc="Message shown when there no NFTs in the NFT grid.">
+      No NFTs here yet.
+    </message>
+    <message name="IDS_ADD_NFTS" desc="How to import NFTs in the NFT grid.">
+      Ready to add some? Just click the button below to import.
+    </message>
+    <message name="IDS_NFT_SUPPORTED_STANDARDS" desc="NFT token standard currently supported.">
+      Note that Brave Wallet currently only supports the ERC721 token standard.
```

Comment:
we also support Solana NFTs so might wanna update this message,  update as per discussion here https://bravesoftware.slack.com/archives/C04DTJT4MDG/p1679914338370169

Reply:
Dropping this entyire string.